packages/flutter_tools/lib/src/application_package.dart

```diff
@@ -186,8 +189,11 @@ class AndroidApk extends ApplicationPackage {
     }
 
     final Iterable<xml.XmlElement> manifests = document.findElements('manifest');
-    if (manifests.isEmpty)
+    if (manifests.isEmpty) {
+      printError('AndroidManifest.xml has no manifest element.');
+      printError('Please check ${manifest?.path ?? 'AndroidManifest.xml'} for errors.');
```

Comment:
Here and below it may be helpful to direct them to the paths they'd find `AndroidManifest.xml` in, or perhaps a documentation page about it.

Reply:
Confirmed that manifest.path should not be null, so they should have a path here and below

I'd like to link to a flutter.dev page, but it seems we don't have one.